static/app/components/sidebar/sidebarAccordion.tsx

```diff
@@ -93,7 +93,11 @@ function findChildElementsInTree(
     }
 
     const currentComponentName: string =
-      typeof child.type === 'string' ? child.type : child.type.name;
+      typeof child.type === 'string'
+        ? child.type
+        : 'displayName' in child.type
+          ? (child.type.displayName as string) // `.displayName` is added by `babel-plugin-add-react-displayname` in production builds
```

Comment:
Is there a way to test logic that will only be present in production builds? I'm still learning 😅

Reply:
You could, but you would probably need to mock the part of the build that adds this, meaning you would really just be testing the mock, which isn't very high value.

Reply:
Thanks! Good to know 👍

Reply:
Yeah, I don't _love_ this solution, TBH, but it's a quicker fix. Another solution is probably to do `instance` of checks against the component itself _but_ I also hate having to import the component in question in order to check against it, and the typing for that fix is gross, blah blah blah